src/Nest.JsonNetSerializer/JsonReaderExtensions.cs

```diff
@@ -20,15 +20,12 @@ public static JToken ReadTokenWithDateParseHandlingNone(this JsonReader reader)
 			return token;
 		}
 
```

Comment:
This is the only functional change I'm making.  Calling the sync version since these older versions of Newtonsoft.Json don't have async support